### PR TITLE
Change grade Label to a Text

### LINKED ISSUE #273: [PE-D][Tester A] Student's attributes cannot be seen if they are too long
The rest of the name cannot be seen, this applies for the other attributes of the student too

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: Dilysss/ped#2

### PR DESCRIPTION
**Pull request by RezwanArefin01**
Fixes #273. 

The grade should not be clipped now. The name will be clipped in part based on available space. 

Maybe we should consider setting a max width for grade #334.

### CODE REVIEW

File: src/main/resources/view/StudentCard.fxml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+
```
Comment (optionalemon) [inline src/main/resources/view/StudentCard.fxml]: maybe delete this line

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -165,14 +165,24 @@
...
 
-.cell_big_label, .titled-pane, .titled-pane > .title .label {
+.titled-pane, .titled-pane > .title Text {
+    -fx-font-family: "Lato";
+    -fx-font-weight: 700;
+    -fx-font-size: 18px;
+    -fx-fill: #144073;
+}
+
+.titled-pane, .titled-pane > .title .label {
```
Comment (optionalemon) [inline src/main/resources/view/DarkTheme.css:175]: `Text` and `label` are mainly the same except the last line. Is it possible to extract the front part out as `.titled-pane, .titled-pane > .title` ? May reduce some redundancies

**Review by optionalemon — CHANGES_REQUESTED** [review]
LGTM! Just some small details to fix :)

**Review by optionalemon — APPROVED** [review]
LGTM!

File: src/main/resources/view/DarkTheme.css (hunk 2 above)
Comment (RezwanArefin01) [reply src/main/resources/view/DarkTheme.css:175]: I tried that before but it does not work as. The `.title` is actually a `HBox` inside `TitledPane`. So need to select the `Text` and `.label` first. Applying to `HBox` doesn't apply to the child.

**Pull request merged**